### PR TITLE
fix: update delete modal

### PR DESCRIPTION
**Pull request by hydr063n**
this fixes the text color in dark mode and the StyledButton is replaced with a StatusButton
For the StatusButton to support the color red, I added the color property that can be overridden

### CODE REVIEW

File: ui/shared/status/StatusButton.qml [hunk 1]
```diff
@@ -8,6 +8,7 @@ Button {
     property string type: "primary"
     property string size: "large"
     property string state: "default"
+    property string color: Style.current.buttonForegroundColor
```
Comment (jrainville) [inline ui/shared/status/StatusButton.qml:11]: The color type is pretty similar to string, but in QT creator, it shows a color picker so it's helpful
```suggestion
    property color color: Style.current.buttonForegroundColor
```

File: ui/shared/ConfirmationDialog.qml [hunk 2]
```diff
@@ -26,6 +27,7 @@ ModalPopup {
 
     Text {
```
Comment (jrainville) [inline ui/shared/ConfirmationDialog.qml:28]: StyledText handled the color and the font so you don't have to
```suggestion
    StyledText {
```

File: ui/shared/ConfirmationDialog.qml [hunk 3]
```diff
@@ -26,6 +27,7 @@ ModalPopup {
...
         text: confirmationDialog.confirmationText
+        color: Style.current.textColor
```
Comment (jrainville) [inline ui/shared/ConfirmationDialog.qml:30]: This isn't needed if you use StyledText instead
```suggestion
```

File: ui/shared/ConfirmationDialog.qml [hunk 4]
```diff
@@ -37,14 +39,11 @@ ModalPopup {
...
-        StyledButton {
+        StatusButton {
             anchors.right: parent.right
             anchors.rightMargin: Style.current.smallPadding
-            btnColor: Style.current.lightRed
-            btnBorderWidth: 1
-            btnBorderColor: Style.current.grey
-            textColor: Style.current.red
-            label: confirmationDialog.confirmButtonLabel
+            color: Style.current.red
```
Comment (jrainville) [inline ui/shared/ConfirmationDialog.qml:45]: `danger` uses `red` but it gives more context to the color use
```suggestion
            color: Style.current.danger
```

**Review by jrainville — COMMENTED** [review]
Nice change. I just added suggestions for a better use of the Style

File: ui/shared/ConfirmationDialog.qml (hunk 3 above)
Comment (hydr063n) [reply ui/shared/ConfirmationDialog.qml]: Dang! Shoulda noticed! Fixed!

File: ui/shared/ConfirmationDialog.qml (hunk 2 above)
Comment (hydr063n) [reply ui/shared/ConfirmationDialog.qml]: Dang! Shoulda noticed that! Fixed!

File: ui/shared/ConfirmationDialog.qml (hunk 4 above)
Comment (hydr063n) [reply ui/shared/ConfirmationDialog.qml]: That makes more sense! Fixed!

**Review by jrainville — APPROVED** [review]
Well done

**Pull request merged**